src/ansible/roles/packages/tasks/Fedora.yml

```diff
@@ -260,7 +282,7 @@
 - name: Install additional packages for client development image
   block:
   - name: Install SSSD build dependencies
-    command: dnf build-dep -y sssd
+    command: dnf builddep -y sssd
```

Comment:
build-dep is an alias that they forgot to implement in dnf5. 
 I raised a issue for it already but this is actual solution that work already.

Reply:
hahaha that's funny because people on the internet usually mention `build-dep`, but not `builddep` so I think they'll have to implement it. Can you link the issue? I'd like to follow it.

Reply:
Added to pr comment.